app/kubernetes/views/deploy/deploy.html

```diff
@@ -118,7 +118,7 @@
                     <editor-description>
                       <span class="col-sm-12 text-muted small" ng-show="ctrl.state.DeployType === ctrl.ManifestDeployTypes.COMPOSE">
                         <p class="vertical-center">
-                          <pr-icon icon="'alert-circle'" mode="'warning'" feather="true"></pr-icon>
+                          <pr-icon icon="'alert-circle'" mode="'warning'"></pr-icon>
```

Comment:
EE has `info` icon. https://github.com/portainer/portainer-ee/pull/2472/files#diff-3e3185001e467389031d8b15861e453bb2894bd8f24c293598613fd3eab687f9R121

Reply:
Warning messages should have 
```icon="'alert-triangle'" mode="'warning'"```
as per https://www.figma.com/file/g5TUMngrblkXM7NHSyQsD1/New-UI?node-id=144%3A2236&t=URpyH2qJo8W6VoKn-0
I've updated cases of 
```icon="'alert-circle'" mode="'warning'"```
to match this. I won't make the additional changes around the text color as it should be covered in a separate ticket